tests/unit/htdocs/Frameworks/art/FunctionsIniTest.php

```
@@ -281,6 +281,69 @@ public function frameworksRootPathIsDefined(): void
+        // mod_loadRenderer will try to require a file, so we need the class pre-loaded
+        // The function checks class_exists() first, so it won't try to require
+        $result = mod_loadRenderer('widget', 'testmod');
+
+        if ($old !== null) {
+            $GLOBALS['xoopsModule'] = $old;
+        } else {
+            unset($GLOBALS['xoopsModule']);
+        }
+
+        $this->assertInstanceOf('TestmodWidgetRenderer', $result);
+    }
```

PHPMD (2.15.0)

[error] modLoadRendererReturnsInstanceFromStaticCall accesses the super-global variable $GLOBALS. (undefined)

(Superglobals)